source/team/ways-of-working.html.md.erb

```diff
@@ -38,3 +42,24 @@ As a team we have agreed to do the following for each stage:
 * __Done__
     * Tasks can be closed when they have been implemented, tested and reviewed.
     * Closing an issue moves a task to __Done__, where moving an issue to __Done__ does not close it, hence close it, rather than move it.
+
+## Working with Git and GitHub
+
+There is a number of things we can do to make collaborating on code seamless.
+Although we do not want to impose too many rules on our team members, the following list is what we agreed as a team to do as a minimum:
+
+* Keeping a good git hygiene `git fetch`/`git pull` in `main` before branching out when starting on a new task. Merging `main` regularly into your branch on local and resolving the merge conflicts before committing and pushing to remote.
+* Adding meaningful messages when committing, such as what have you achieved in this step.
+* Running tests, verifying if the tests/SCA has passed and that the `terraform plan` and the pipeline runs clean before creating a PR.
+* Providing enough information in the PR description so that another team member has enough context to be able to understand the change and to review it.
+Adding explanation on what is implemented with the PR and why. Adding information on tests if they were run outside of the GitHub actions pipeline and other useful links or references that help to understand the changes (e.g. a link to the issue or a doc).
```

Comment:
Is this line missing a `*`?

Reply:
it's a continuation of the previous line